### PR TITLE
tighten up buildbox config and fix modulepath

### PR DESCRIPTION
**Pull request by phrawzty**
@rhelmer `r?`

### CODE REVIEW

File: puppet/modules/socorro/manifests/role/buildbox.pp [hunk 1]
```diff
@@ -39,7 +39,7 @@
   file {
     'deadci.ini':
       ensure  => file,
-      path    => '/var/lib/deadci.ini',
+      path    => '/var/lib/deadci/deadci.ini',
```
Comment (phrawzty) [inline puppet/modules/socorro/manifests/role/buildbox.pp]: :blush:

File: terraform/socorro_role.sh [hunk 2]
```diff
@@ -18,7 +18,7 @@ function socorro_role {
     # Provision the role.
     /usr/bin/env FACTER_socorro_role=$2 \
         puppet apply \
-        --modulepath=${DIR}/puppet/modules \
+        --modulepath=${DIR}/puppet/modules:/etc/puppet/modules \
```
Comment (phrawzty) [inline terraform/socorro_role.sh]: We need both *our* modules as well as those in the system.

File: terraform/variables.tf [hunk 3]
```diff
@@ -20,6 +20,11 @@ variable "base_ami" {
         us-west-2 = "ami-51604e61"
     }
 }
+variable "buildbox_ami" {
```
Comment (phrawzty) [inline terraform/variables.tf]: Time to start tracking this, too.

File: terraform/main.tf [hunk 4]
```diff
@@ -471,28 +481,28 @@ resource "aws_elb" "elb_for_buildbox" {
...
     instance_type = "t2.micro"
     key_name = "${lookup(var.ssh_key_name, var.region)}"
     security_groups = [
-        "${aws_security_group.internet_to_elb__http.name}",
-        "${aws_security_group.elb_to_buildbox__http.name}",
+        "${aws_security_group.elb_to_buildbox__deadci.name}",
         "${aws_security_group.internet_to_any__ssh.name}",
         "${aws_security_group.private_to_private__any.name}"
     ]
+    iam_instance_profile = "buildbox"
```
Comment (phrawzty) [inline terraform/main.tf]: This is the magic right here. We'll need to apply `iam_instance_profile` throughout.  By default we'll use the `default` role. :wine_glass:

**Comment by rhelmer** [comment]
r+

**Pull request merged**